src/dom.js

```diff
@@ -541,6 +542,8 @@ function scopedQuerySelectionFallback(root, selector) {
  * @param {!Element|!ShadowRoot} root
  * @param {string} selector
  * @return {?Element}
+ *
+ * @suppress {suspiciousCode}
```

Comment:
What is this suppressing?

Reply:
Any function which has a build-time constant like `IS_ESM` becomes suspicious code:
![Screen Shot 2021-04-29 at 11 37 52 AM](https://user-images.githubusercontent.com/4656974/116578399-5f00e680-a8df-11eb-9923-6c49fbada557.png)

Reply:
Not for this PR, but we should resolve this by providing a helper like we do in `src/core/minified-mode.js` and then ban using `IS_ESM` anywhere